config/secrets.yml

```diff
@@ -0,0 +1,10 @@
+RAILS_MAX_THREADS: "5"
```

Comment:
This file should be in .gitignore and not committed to github. Could create a secrets.yml.sample file, with dummy values (for example, no server name like houston in the value). 

And add instruction to the readme to copy and modify the sample file